### PR TITLE
Documenting Equipment API

### PR DESCRIPTION
**Pull request by bltavares**
screen shot 2016-04-29 at 10 10 17 am

### CODE REVIEW

**Comment by glauco** [comment]
LGTM!

File: source/includes/_equipments.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+404  | Not Found
+500  | Unhandled Error
+
+
+### 200 Successful
+
+> The returned model schema is the following:
+
+```json
+[[equipments:model:Model 2]]
```
Comment (jeffhsta) [inline source/includes/_equipments.md]: In this line, what the number 2 means?

Comment (bltavares) [inline source/includes/_equipments.md]: This is a `Model 2` reference as it is showing on the Swagger response for our endpoint.

Comment (jeffhsta) [inline source/includes/_equipments.md]: :+1:

**Pull request merged**